src/CarbonSystemSolvers.jl

```diff
@@ -12,10 +12,167 @@ struct CarbonSystem{FT}
     pCO₂ᵃᵗᵐ:: FT
 end
 
+module CarbonSolverFollows
+export CarbonSystemFollows
```

Comment:
Should this be "FollowsCarbonSystem" (following Oceananigans naming style)? The convention is to name variables as they are spoken aloud.

Reply:
We also usually try to avoid using people's names if possible, if a more explicit description is available

Reply:
Fair enough, I'll give it some thought. Perhaps `CarbonSolverFollows` -> `CarbonSolverIterativeAlkalinityCorrection` and `CarbonSolverMunhoven` -> `CarbonSolverRobustRational`, or something like that?